docs/csaf_aggregator.md

```diff
@@ -111,11 +111,13 @@ in a `aggregator.category == "aggregator"` instance,
 set `category` to `lister` in the entry.
 Otherwise it is recommended to not set `category` for entries.
 
-If a provider's domain starts with https:// its considered a publisher.
-These publishers are added to the csaf_publishers list instead of the csaf_providers list which was used before.
-
-The mandatory value of update_interval can be configured for each publisher (provider) individually. If not given it falls back
-to the global update_interval. If this is not given either the default is "on best effort".
+If a provider's domain starts with `https://` it is considered a publisher.
+These publishers are added to the `csaf_publishers` list, written
+to the resulting `aggregator.json`.
+Each publisher must announce an `update_interval` there.
+This can be configured for each entry, by the config option with the same name.
+If not given it is taken from the configured default
```

Comment:
I had removed the "global" because I find it slightly ambiguous. The https://toml.io/en/v1.0.0 does not use this term for example and this is the general config section, so I tried to resolve this by contrasting it with the configuration "for each entry". 

We need to reconsider this again, when we fix the problems mentioned #317 because `rate` is handled similiar. It can be better to just explain the concept once for both. So I suggest to but this mergerequest in as an improvement.

Reply:
put it in, as it is now...

Reply:
Ok. I understand the reasoning. I'm going to resolve that here and add a comment in #317. The more often I read it, I think the the distinction might even be clear without the "global".